nncf/openvino/quantization/quantize_model.py

```diff
@@ -438,9 +439,32 @@ def quantize_with_accuracy_control_impl(
     )
 
 
-def compress_weights_impl(model: ov.Model) -> ov.Model:
+def compress_weights_impl(
+    model: ov.Model,
+    mode: CompressWeightsMode = CompressWeightsMode.COMPRESSED_INT8,
+    ratio: float = None,
+    group_size: int = None,
+) -> ov.Model:
     """
     Implementation of the `compress_weights()` method for the OpenVINO backend.
+
+    :param model: an OpenVINO model for compression.
+    :param mode: Defines a mode for weight compression.
+        COMPRESSED_INT8 stands for unsigned int8 quantization of all weights.
```

Comment:
Taking into account a previous comment from @l-bat, you can say  "...8-bit integer quantization..." instead of "int8".

Reply:
Thanks, corrected!